### PR TITLE
CIV-7123  Welsh translation

### PR DESCRIPTION
**Pull request by pascalkankonde**
### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-7123

### Change description ###
The scope of this ticket is to develop Welsh content translation of the following screens: 

DQ- Welsh Language
DQ - Why are you, your experts or witnesses unavailable for a hearing for <xx> days
 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/main/modules/i18n/locales/cy.json [hunk 1]
```diff
@@ -2116,23 +2116,23 @@
       "TITLE": "A ydych chi, eich arbenigwyr neu eich tystion yn agored i niwed mewn ffordd y mae angen i’r llys ei ystyried?"
     },
     "WELSH_LANGUAGE": {
```
Comment (pascalkankonde) [inline src/main/modules/i18n/locales/cy.json:2118]: THIS SHOULD BE REFACTORED (After a conversation the person who implemented this and tech lead)

**Review by neeta-hmcts — APPROVED** [review]

File: src/main/modules/i18n/locales/cy.json [hunk 2]
```diff
@@ -334,8 +334,8 @@
       "PAGE_TITLE": "Do you want to respond to this claim in Welsh?",
       "TITLE": "Do you want to respond to this claim in Welsh?",
       "DESCRIPTION": "You can respond to this claim in Welsh. After you've submitted your response, anything else you may have to do relating to the claim will be in English. You'll receive emails and letters in English.",
-      "ENGLISH": "English",
-      "WELSH_AND_ENGLISH": "Welsh and English"
+      "ENGLISH": "Cymraeg",
```
Comment (laperla11) [inline src/main/modules/i18n/locales/cy.json:337]: `Cymraeg` means Welsh not `English`?

File: src/main/modules/i18n/locales/cy.json [hunk 3]
```diff
@@ -36,8 +36,8 @@
     "REASON": "Rheswm",
     "REMOVE": "Tynnu",
     "RETURN_PAGE": "You can return to RETURN-PAGE and start your claim or defence again.",
-    "TECHNICAL_PROBLEMS": "Sorry, we had some technical problems during your last operation. Please try again later.",
     "TOTAL_DEBT": "Cyfanswm sy’n ddyledus gennych",
+    "TECHNICAL_PROBLEMS": "Sorry, we had some technical problems during your last operation. Please try again later.",
```
Comment (laperla11) [inline src/main/modules/i18n/locales/cy.json:40]: There is no translation herfe only changing lines?

File: src/main/modules/i18n/locales/cy.json [hunk 4]
```diff
@@ -2137,23 +2137,23 @@
...
     },
     "WELSH_LANGUAGE": {
-      "ENGLISH": "English",
-      "OFFICIAL_LANGUAGE": "Welsh is an official language of Wales. You can use Welsh in court hearings. Asking to speak in Welsh in your hearing will not delay the hearing or have any effect on proceedings or the outcome of a case.",
-      "PAGE_TITLE": "Welsh language",
-      "WELSH": "Welsh",
-      "WELSH_AND_ENGLISH": "Welsh and English",
-      "WHAT_LANGUAGE_DOCUMENTS": "What languages will the documents be provided in?",
-      "WHAT_LANGUAGE_SPEAK": "What languages will you, your experts and your witnesses speak at the hearing?"
+      "ENGLISH": "Cymraeg",
```
Comment (laperla11) [inline src/main/modules/i18n/locales/cy.json:2140]: same here

File: src/main/modules/i18n/locales/cy.json [hunk 5]
```diff
@@ -2137,23 +2137,23 @@
...
+      "OFFICIAL_LANGUAGE": "Mae’r Gymraeg yn iaith swyddogol yng Nghymru. Gallwch ddefnyddio’r Gymraeg mewn gwrandawiad llys. Ni fydd gofyn am gael siarad Cymraeg yn eich gwrandawiad yn achosi oedi wrth ei drefnu neu’n cael unrhyw effaith ar yr achos neu ganlyniad yr achos.",
+      "PAGE_TITLE": "Yr iaith Gymraeg",
+      "WELSH": "Saesneg",
```
Comment (laperla11) [inline src/main/modules/i18n/locales/cy.json:2143]: `Saeseng` means English in Welsh language

**Review by laperla11 — CHANGES_REQUESTED** [review]

**Comment by PaulH-HMCTS** [comment]
Can you ensure that comments left on previous PRs have been answered and addressed please? For example https://github.com/hmcts/civil-citizen-ui/pull/1363#pullrequestreview-1299625096

File: src/main/modules/i18n/locales/cy.json (hunk 2 above)
Comment (PaulH-HMCTS) [reply src/main/modules/i18n/locales/cy.json:337]: As expected Smily has confirmed these changes should be made:
"ENGLISH":  "Saesneg"
"WELSH": "Cymraeg"

File: src/main/modules/i18n/locales/cy.json (hunk 5 above)
Comment (laarmada) [reply src/main/modules/i18n/locales/cy.json:2143]: fix

File: src/main/modules/i18n/locales/cy.json (hunk 4 above)
Comment (laarmada) [reply src/main/modules/i18n/locales/cy.json:2140]: fix and review hole document

File: src/main/modules/i18n/locales/cy.json (hunk 2 above)
Comment (laarmada) [reply src/main/modules/i18n/locales/cy.json:337]: modify and reviewed

File: src/main/modules/i18n/locales/cy.json (hunk 3 above)
Comment (laarmada) [reply src/main/modules/i18n/locales/cy.json:40]: this key is not related with the ticket pages

**Review by laarmada — APPROVED** [review]

**Review by laperla11 — APPROVED** [review]

**Review by LeonardoPalmeiro — APPROVED** [review]

File: src/main/modules/i18n/locales/cy.json [hunk 6]
```diff
@@ -413,7 +413,7 @@
       "CLAIM_INTEREST": "Claim Interest",
       "CONTACT_PERSON": "Cyswllt",
       "CONTACT_NUMBER": "Rhif cyswllt (dewisol)",
-      "CONSIDER_CLAIMANT_DOCUMENT" : "Are there any documents the claimant has that you want the court to consider?",
+      "CONSIDER_CLAIMANT_DOCUMENT": "Are there any documents the claimant has that you want the court to consider?",
```
Comment (GorilaNaranja) [inline src/main/modules/i18n/locales/cy.json:416]: Just to be sure, are not these translations in the scope? It is just because the format right?

**Review by TatsyMaslova — APPROVED** [review]

File: src/main/modules/i18n/locales/cy.json (hunk 6 above)
Comment (laarmada) [reply src/main/modules/i18n/locales/cy.json:416]: you are right only format

**Review by GorilaNaranja — APPROVED** [review]

**Review by LeonardoPalmeiro — APPROVED** [review]

**Review by PaulH-HMCTS — APPROVED** [review]

**Comment by laarmada** [comment]
not valid user

**Pull request closed without merging**